cmd/signaling/main.go

```diff
@@ -57,7 +57,6 @@ func main() {
 	corsHandler := cors.Default()
 	handler := corsHandler.Handler(mux)
 	handler = util.NoStoreMiddleware(handler)
-	handler = logging.Middleware(handler, logger)
```

Comment:
This will completely remove all logging, that's not what we want.
This change should be removed and this should be merged: https://github.com/poki/netlib/pull/279